app/screens/map-screen/map-screen.tsx

```diff
@@ -41,6 +41,7 @@ type Props = {
 }
 
 gql`
+  # TODO: caching
```

Comment:
We can try defining a type policy for this in the cache using the coordinates as a key identifier since they're unique.
Something like
`typePolicies: {
   (typeof buniessMapMarkers): {
           keyFields: [ "coordinates", ["longitude", "latitude"] ]
       }
}`